app/src/org/commcare/activities/FormEntryActivityUIController.java

```diff
@@ -562,6 +573,10 @@ protected void next() {
         }
     }
 
+    protected boolean shouldIgnoreNavigationAction() {
```

Comment:
I was a bit surprised to see this get created v. added to shouldIgnoreSwipeAction, seems like they are accomplishing the same thing, right? Shouldn't this just get added to the list of reasons a "nav action" gets ignored?